frontend/src/components/CommandPalette.tsx

```diff
@@ -0,0 +1,190 @@
+            <Send />
+            <span>Send Payment</span>
+          </CommandItem>
+          <CommandItem
+            onSelect={() => runCommand(() => navigate("/wallet/receive"))}
+          >
+            <QrCode />
+            <span>Receive Payment</span>
+          </CommandItem>
+          <CommandItem
+            onSelect={() => runCommand(() => navigate("/wallet/swap"))}
+          >
+            <Shuffle />
+            <span>Swap</span>
+          </CommandItem>
+          <CommandItem
+            onSelect={() =>
+              runCommand(() => navigate("/wallet/receive/invoice"))
+            }
+          >
+            <FileText />
+            <span>Create Invoice</span>
+          </CommandItem>
+          <CommandItem
+            onSelect={() =>
+              runCommand(() => navigate("/wallet/receive/onchain"))
+            }
+          >
+            <Zap />
+            <span>Receive On-chain</span>
+          </CommandItem>
+          <CommandItem
+            onSelect={() => runCommand(() => navigate("/wallet/sign-message"))}
+          >
+            <Shield />
+            <span>Sign Message</span>
+          </CommandItem>
+        </CommandGroup>
+        <CommandSeparator />
+        <CommandGroup heading="Settings">
+          <CommandItem onSelect={() => runCommand(() => navigate("/settings"))}>
+            <Settings />
+            <span>Settings</span>
+          </CommandItem>
+          <CommandItem
+            onSelect={() => runCommand(() => navigate("/settings/backup"))}
+          >
+            <Shield />
+            <span>Backup</span>
+          </CommandItem>
+          <CommandItem
+            onSelect={() =>
+              runCommand(() => navigate("/settings/alby-account"))
+            }
+          >
+            <User />
+            <span>Alby Account</span>
+          </CommandItem>
+          <CommandItem
+            onSelect={() => runCommand(() => navigate("/settings/about"))}
+          >
+            <Info />
+            <span>About</span>
+          </CommandItem>
+          <CommandItem
+            onSelect={() => runCommand(() => navigate("/settings/developer"))}
+          >
+            <HelpCircle />
+            <span>Developer Settings</span>
+          </CommandItem>
+        </CommandGroup>
+        <CommandSeparator />
+        <CommandGroup heading="Quick Actions">
+          <CommandItem onSelect={() => runCommand(() => navigate("/apps/new"))}>
+            <Plug />
+            <span>Connect New App</span>
+          </CommandItem>
+          <CommandItem
+            onSelect={() => runCommand(() => navigate("/sub-wallets/new"))}
+          >
+            <CreditCard />
+            <span>Create Sub-wallet</span>
+          </CommandItem>
+          <CommandItem
+            onSelect={() => runCommand(() => navigate("/peers/new"))}
+          >
+            <Users />
+            <span>Connect Peer</span>
+          </CommandItem>
+          <CommandItem
```

Comment:
I think we should remove this one or at least only show it if the user doesn't have a channel :thinking:

Reply:
there's no entry for opening another channel